javascript/ql/experimental/adaptivethreatmodeling/src/codeql-suites/javascript-atm-code-scanning.qls

```diff
@@ -1,8 +1,2 @@
 - description: ATM boosted Code Scanning queries for JavaScript
 - queries: .
-- include:
-    id: 
-    - adaptive-threat-modeling/js/nosql-injection
-    - adaptive-threat-modeling/js/sql-injection
-    - adaptive-threat-modeling/js/path-injection
-    - adaptive-threat-modeling/js/xss
```

Comment:
Why has this been removed in this PR?

Reply:
Yeah arguably that commit should have been in a separate PR.  The justification is:

> It's simpler to just run all the queries in the pack instead of specifying the IDs.